sdk/search/Azure.Search.Documents/src/Models/Answers.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Search.Documents.Models
+{
+    internal readonly partial struct Answers
```

Comment:
Just an FYI that we do make these public soon we'll need more specific names than Answers, etc.

Reply:
Yes, I have the models names as `QueryAnswer` (no plural and less generic) and `QuerySpeller` (less generic).